camera_name: cascade/milestone_universal_camera
frame_id: cascade/milestone_universal_camera/camera_link
settings:
  framerate: 1
  manual_acquisition: False
uri:
  ip: "192.168.1.20"
  protocol: "rtsp"
  port: "8554"
  username: ""
  password: ""
  suffix: ""
  device_index: 0
